en/docs/get-started/quick-start-guide.md

```diff
@@ -85,8 +85,24 @@ using WSO2 Identity Server.
     server.host.domain=localhost.com
     server.host.port=8080
     ```
+3.  Add the following configurations to the deployment.toml file in `<IS_HOME>/repository/conf` directory to make the CORS configurations applicable.
+
+    ``` toml
+    [cors]
+    allow_generic_http_requests = true
+    allow_any_origin = true
+    supported_methods = [
+        "POST",
+        "HEAD",
+        "OPTIONS"
+    ]
+    supports_credentials = true
```

Comment:
Check whether we can keep this option false

Reply:
Addressed with https://github.com/wso2/docs-is/pull/2813/commits/9f56951744c587262dcc392faa904b2d7bbac46d